vijaya/Table_with_string

```diff
@@ -0,0 +1,17 @@
+s = "Hi Team, the question is given by me & it should be optimizable code & the time complexity & space complexity should be equal or less than O(n)."
+t = 2
+s1 = ""
+for i in s:
+    if (ord(i) >= 97 and ord(i) <= 122) or (ord(i) >= 65 and ord(i) <= 90):
+        s1 += i
+print(s1)
+#  to find the length of the given string
+length = 0
+for j in s1:
```

Comment:
why this loop here?

Reply:
As I mentioned in the code as comment , that loop is to find the length of the string